scripts/index.js

```diff
@@ -36,6 +30,6 @@ window.addEventListener("online", () => {store.online = true;});
 
 // Make sure local data depend on current user.
 // Note: Kinto.js will have an option: https://github.com/Kinto/kinto.js/pull/111
-store.collection.db.dbname = userpass64 + store.collection.db.dbname;
+store.collection.db.dbname = auth.token + store.collection.db.dbname;
```

Comment:
First I wish we could use last kinto version with prefix parameter.
Then this means no prefix for FXA? (It might not be a problem though)